### ISSUE TITLE
Use TDR to read metrics. 

### ISSUE THREAD

**Issue by lynnpais** [broadinstitute/seqr#4292]
**Is your feature request related to a problem? Please describe.**
Reduce manual work during the data loading process.

**Describe the solution you'd like**
Read metrics directly from TDR. 

**Describe alternatives you've considered**
NA
**Additional context**
Based on this morning's meeting. PMs will follow up on getting access to the TDR.

**Comment by bpblanken** [comment]
@lynnpais Is it possible to track down an API route that I should use to request the file?

**Comment by lynnpais** [comment]
@shall48 would you be able to answer Ben's question or know someone in the TDR team to approach?

**Comment by lynnpais** [comment]
@bpblanken so that you are not blocked indefinitely, I looked into this a bit. Is any of this information helpful/ relevant? 

How to ingest and update TDR data with APIs - https://support.terra.bio/hc/en-us/articles/4407166231963-How-to-ingest-and-update-TDR-data-with-APIs

How to create a TDR dataset with APIs - https://support.terra.bio/hc/en-us/articles/4407157465115-How-to-create-a-TDR-dataset-with-APIs

**Comment by bpblanken** [comment]
@lynnpais yes this does look helpful!  I will take a look this afternoon to see if this can get us what we need.

**Comment by shall48** [comment]
@lynnpais and @bpblanken I have not heard back on a good person to reach out to, but terra support may be able to help.
